.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
@VAlmea Should this file be included, or should the `.vscode` folder be ignored?

Reply:
This file is used to debug the generator in development mode. Maybe it would be a better idea to update the documentation

Reply:
Yes. We shouldn't commit it, and instead, create documentation to tell people how to do this themselves.  We'll create a page in the Wiki for this.